ide/include.am

```diff
@@ -2,10 +2,11 @@
 # included from Top Level Makefile.am
 # All paths should be given relative to the root
 
-include ide/winvs/include.am
 include ide/CSBENCH/include.am
-include ide/MQX/include.am
+incldue ide/Espressif/include.am

```

Comment:
This is what is breaking the builds, typo

Reply:
oh! thanks so much for that. fixed